docs/source/using-federation/apollo-subgraph-setup.mdx

```diff
@@ -0,0 +1,351 @@
+---
+title: Implementing a subgraph with Apollo Server
+---
+
+This article demonstrates how to create a **subgraph** for a federated supergraph using Node.js and Apollo Server.
```

Comment:
[This article is pretty much exactly the same](https://www.apollographql.com/docs/federation/building-supergraphs/subgraphs-apollo-server). I just fixed the links :)